en/core-libraries/time.rst

```diff
@@ -37,77 +37,100 @@ For more details on Chronos please see `the API documentation
 
 .. start-time
 
-Creating Time Instances
+Creating FrozenTime Instances
 =======================
 
+``FrozenTime`` are immutable objects that are useful when you want to prevent
+accidental changes to data, or when you want to avoid order based dependency
+issues. Refer to ``Time`` instances for mutable objects.
+
 There are a few ways to create ``FrozenTime`` instances::
 
     use Cake\I18n\FrozenTime;
 
     // Create from a string datetime.
     $time = FrozenTime::createFromFormat(
         'Y-m-d H:i:s',
-        $datetime,
+        '2021-01-31 22:11:30',
         'America/New_York'
     );
 
-    // Create from a timestamp
-    $time = FrozenTime::createFromTimestamp($ts);
+    // Create from a timestamp and set timezone
+    $time = FrozenTime::createFromTimestamp(1612149090, 'America/New_York');
 
     // Get the current time.
     $time = FrozenTime::now();
 
     // Or just use 'new'
-    $time = new FrozenTime('2014-01-10 11:11', 'America/New_York');
+    $time = new FrozenTime('2021-01-31 22:11:30', 'America/New_York');
 
     $time = new FrozenTime('2 hours ago');
 
 The ``FrozenTime`` class constructor can take any parameter that the internal ``DateTimeImmutable``
 PHP class can. When passing a number or numeric string, it will be interpreted
 as a UNIX timestamp.
 
-In test cases you can mock out ``now()`` using ``setTestNow()``::
+In test cases, you can mock out ``now()`` using ``setTestNow()``::
 
     // Fixate time.
-    $now = new FrozenTime('2014-04-12 12:22:30');
-    FrozenTime::setTestNow($now);
+    $time = new FrozenTime('2021-01-31 22:11:30');
+    FrozenTime::setTestNow($time);
 
-    // Returns '2014-04-12 12:22:30'
+    // Outputs '2021-01-31 22:11:30'
     $now = FrozenTime::now();
+    echo $now->i18nFormat('yyyy-MM-dd HH:mm:ss');
 
-    // Returns '2014-04-12 12:22:30'
+    // Outputs '2021-01-31 22:11:30'
     $now = FrozenTime::parse('now');
+    echo $now->i18nFormat('yyyy-MM-dd HH:mm:ss');
 
 Manipulation
 ============
 
-Once created, you can manipulate ``FrozenTime`` instances using setter methods::
+Once created, ``FrozenTime`` instances cannot be manipulated or changed anymore, as they are immutable. Manipulating them will create new instance::
```

Comment:
This sounds like Schrödinger's time objects again, as if you can, and can not manipulate them.

Reply:
How about this  :

Once created, ``FrozenTime`` instances cannot be manipulated or changed, as it is immutable. A new instance can be created from a ``FrozenTime`` instance.